src/NuGet.Config

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8?>
+<configuration>
+  <packageSources>
+    <add key="NuGet Official v2 Package Source" value="https://www.nuget.org/api/v2/" />

```

Comment:
You probably should also add `<clear />` before adding the new key.
Otherwise, this would only add the key to the existing sources on the developers machen.

Reply:
@AdmiringWorm I actually debated about adding the `<clear/>` or not.  In the test machine that I set up for testing this out, it wasn't needed, so I opted to not include it.  I was reliably able to make the build fail until I made the change in this PR.

Reply:
I just remember that it caused issues for me in the past (several years ago I believe, just around the time nuget started using the V3 api).

If it isn't needed anymore, then that is just great.

Reply:
Clear, could increase perf. as it would skip non working feeds.

Reply:
I think we'll want to ensure maximum compatibility in this case. Great suggestions.